packages/shared/src/graphql/posts.ts

```diff
@@ -100,6 +101,7 @@ export const POST_BY_ID_QUERY = gql`
 `;
 
 export const POST_UPVOTES_BY_ID_QUERY = gql`
+  ${UPVOTER_FRAGMENT}
   query PostUpvotes($id: String!, $after: String, $first: Int) {
     postUpvotes(id: $id, after: $after, first: $first) {
```

Comment:
If you want you can do this on both queries:
```suggestion
    upvotes: postUpvotes(id: $id, after: $after, first: $first) {
```
Then it will rename the result key to `upvotes` and you can keep it consistent throughout your requests instead of passing the result key.
But whatever makes more sense, I don't care.

Reply:
Oh, I see. I think that is much simpler.